BenchBnb/BenchBnb/Models/Review.cs

```diff
@@ -8,7 +10,16 @@ namespace BenchBnb.Models
     public class Review
     {
         public int Id { get; set; }
-        float Rating { get; set; }
-        string Comment { get; set; }
+        [Required, Column("Rating")]
+        public float Rating { get; set; }
+        [Required, Column("Comment")]
+        public string Comment { get; set; }
+
+        public Review(float rating, string comment)
```

Comment:
Shouldn't a review be for a specific bench?

Reply:
Fixed